wolox-books/src/app/screens/auth/screens/book-list/book-list.component.html

```diff
@@ -11,17 +11,19 @@
   </div>
 
   <div class="book-list">
-    <a
-      class="book-link"
-      *ngFor="let book of books | searchPipe: searchTitle; trackBy: trackBook"
-      (click)="openDetail(book.id)"
-    >
+    <a class="book-link" *ngFor="let book of books | searchPipe: searchTitle; trackBy: trackBook">
       <div class="book">
-        <img [src]="book.image_url" alt="book cover" class="book__cover" />
-        <div class="book__info">
+        <img
+          [src]="book.image_url"
+          alt="book cover"
+          class="book__cover"
+          (click)="openDetail(book.id)"
```

Comment:
I think this should still be a link and instead of a method, should be a router link